### DISCUSSION TITLE
CARE DRM

### DISCUSSION THREAD

**Discussion by gambr** [apprenticeharper/DeDRM_tools#1909] [Q&A]
I cant find a way to remove CARE DRM from a epub to move the book to a kindle that does not support lcpl original file format.
Is it not supporter yet in DeDRM?

Regards

**Comment by ElleKayEm** [comment]
No, and unlikely to be since as far as I know it's only used by libraries.

**Reply by gambr** [reply]
I do not understand why a DeDRM tool should not handle a DRM variant just because it is used by libraries. In my case is correct, it come from a library, but i honestly dont know if this is the same in other countries.

**Reply by ElleKayEm** [reply]
DeDRM is intended for books you own, not for those on loan.

**Reply by Leseratte10** [reply]
Readium LCP DRM (which is the actual spec, CARE is just an LCP implementation of one particular vendor - in this case, Vivlio) is not only used in libraries. I found multiple book stores that offer some of their books with LCP DRM, too. 

Examples, Bookeen (https://www.bookeenstore.com/, you can even filter for LCP-eBooks only), ePagine (french book distribution platform, https://lcp.epagine.fr/ ) or Gallimard (large french publisher, https://lcp.gallimard.fr/en ) are all offering eBooks for sale (not as a temporary loan like in a library) with the books encrypted with LCP. 

Yeah, right now you'll be able to buy all these books with Adobe DRM, too (by just going to a different store), but who knows how long that'll be the case. You never know if french publishers (LCP seems to be vastly more popular in France than in other countries) decide to switch to LCP because it's uncracked so far ...

**Reply by ElleKayEm** [reply]
When a DRM scheme becomes more unavoidable that's usually when people with the skills to figure it out get motivated to do so.

**Reply by def050984-canada** [reply]
Hi
First and foremost, You did really good work. Congratulations. My message is just an update. Indeed (https://banq.pretnumerique.ca) a Canadian library works with the LCP DRM 3 month ago.  The LCP DRM list is growing up. Would you please reconsider your point of view ?

**Reply by ElleKayEm** [reply]
noDRM added support for it, but had to remove it because Readium issued a DMCA takedown.  See this:  https://github.com/noDRM/DeDRM_tools/issues/18

**Comment by noDRM** [comment]
like this fork?
https://github.com/noDRM/DeDRM_tools
